Capgemini.Xrm.CdsDataMigrator/Capgemini.Xrm.CdsDataMigratorLibrary.Tests.Unit/Presenters/ExportPresenterTests.cs

```diff
@@ -92,7 +93,7 @@ public void ExportDataActionThrowsException()
                                 .Throws<OrganizationalServiceException>();
             logger.Setup(a => a.LogError(It.IsAny<string>()));
 
-            FluentActions.Invoking(() => systemUnderTest.ExportDataAction())
+            FluentActions.Invoking(() => systemUnderTest./*ExportDataAction*/ExportData(null, new EventArgs()))
```

Comment:
Can this comment be removed? 
```suggestion
            FluentActions.Invoking(() => systemUnderTest.ExportData(null, new EventArgs()))
```